airbyte-integrations/connectors/source-oracle-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/oracle_strict_encrypt/AirbyteOracteTestContainer.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.oracle_strict_encrypt;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static java.util.Collections.singleton;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.Future;
+import org.apache.commons.lang3.StringUtils;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+public class AirbyteOracteTestContainer extends JdbcDatabaseContainer<AirbyteOracteTestContainer> {
```

Comment:
This class is same as `OracleContainer`, why do we need both the classes?

Reply:
Hmmm... this seems to be a merge-conflict issue. That had to be a renaming rather than new class creation. Updated